sdcard/LOOKINGGLASS/PRESETS.TXT

```diff
@@ -25,6 +25,8 @@
 1879,1931,DECT
 # FRS/GMRS
 462,468,FRS/GMRS
+# AUS UHF
+476,478,AUS UHF
```

Comment:
If this frequency range is specific to Australian UHF CB, I would propose calling it "AUS UHF CB" for a more specific description.

Reply:
Thanks for the recommendation, I've updated it accordingly in the PR.